azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participant/grid/cell/ParticipantGridCellVideoView.kt

```diff
@@ -64,40 +70,88 @@ internal class ParticipantGridCellVideoView(
     }
 
     private fun updateVideoStream(
-        it: VideoViewModel?,
+        videoViewModel: VideoViewModel?,
     ) {
         if (videoStream != null) {
             detachFromParentView(videoStream)
             videoStream = null
         }
 
-        if (it != null) {
-            getVideoStream(
+        if (videoViewModel != null) {
+            getVideoStreamCallback(
                 participantViewModel.getParticipantUserIdentifier(),
-                it.videoStreamID
+                videoViewModel.videoStreamID
             )?.let { view ->
                 videoStream = view
-                setRendererView(view)
+                setRendererView(view, videoViewModel.streamType)
             }
+        } else {
+            removeScreenShareZoomView()
+        }
+    }
+
+    private fun removeScreenShareZoomView() {
+        if (screenShareZoomFrameLayout != null) {
+            // removing this code will cause issue when new user share screen (zoom will not work)
```

Comment:
we already have `screenShareZoomFrameLayout.enableZoom()`, so would adding `screenShareZoomFrameLayout.disableZoom()` fix the problem?

Reply:
No, we need to remove view to make sure. it is no more in parent view.